base-notebook/jupyter_notebook_config.py

```diff
@@ -28,28 +28,42 @@
             raise
 
     # Generate an openssl.cnf file to set the distinguished name
-    cnf_file = os.path.join(os.getenv('CONDA_DIR', '/usr/lib'), 'ssl', 'openssl.cnf')
+    cnf_file = os.path.join(os.getenv("CONDA_DIR", "/usr/lib"), "ssl", "openssl.cnf")
     if not os.path.isfile(cnf_file):
-        with open(cnf_file, 'w') as fh:
-            fh.write('''\
+        with open(cnf_file, "w") as fh:
+            fh.write(
+                """\
 [req]
 distinguished_name = req_distinguished_name
 [req_distinguished_name]
-''')
+"""
+            )
 
     # Generate a certificate if one doesn't exist on disk
-    subprocess.check_call(['openssl', 'req', '-new',
-                           '-newkey', 'rsa:2048',
-                           '-days', '365',
-                           '-nodes', '-x509',
-                           '-subj', '/C=XX/ST=XX/L=XX/O=generated/CN=generated',
-                           '-keyout', pem_file,
-                           '-out', pem_file])
+    subprocess.check_call(
+        [
+            "openssl",
+            "req",
+            "-new",
+            "-newkey",
+            "rsa:2048",
+            "-days",
+            "365",
+            "-nodes",
+            "-x509",
+            "-subj",
+            "/C=XX/ST=XX/L=XX/O=generated/CN=generated",
+            "-keyout",
+            pem_file,
+            "-out",
+            pem_file,
+        ]
+    )
```

Comment:
Will do, in this case one can use `f"-out={pem_file}"` for example and such to avoid breaking apart flags that now have key/value across two separate lines.

Reply:
If it works, then yes, it's a good solution